1_provision/README.md

```diff
@@ -68,6 +68,10 @@ $ source ../configuration_parameters_common/tf_vars.sh
 $ terraform destroy
 ```
 
+> **_NOTE:_**  
+> Before you can ssh into your running instances, you will need to upload the public ssh key `../configuration_parameters_common/vm_ssh_key.pub`.  
```

Comment:
I prefer "SSH" in documentation to be capitalized (which is also the standard in the OCI docs; i.e. "SSH" not "ssh").

I'm not sure what you mean by "need to upload the public ssh key". Upload to where?

Reply:
On the OCI platform, before I could SSH into an instance, I needed to upload the public key via Compute -> Instances -> Instance Details -> Console Connection. Before doing that the connection would timeout. There may be a more efficient way to do it via the CLI, but I couldn't find the reference.